api/src/main/java/org/openmrs/module/muzimafingerPrint/PatientJsonParser.java

```diff
@@ -45,6 +45,7 @@ public Encounter CreateEncounter(String patientData) throws JSONException, Parse
     public MuzimaFingerprint CreatePatientFingerPrint(Patient patient, String patientData) throws JSONException {
 
         MuzimaFingerprint fingerprint = new MuzimaFingerprint();
+        System.out.println("In create patient finger print "+patientData);
```

Comment:
We probably want to use logger.log to print out debug message instead of system.out.